generate-packaging-dir

```diff
@@ -13,5 +13,5 @@ set -eu
 
 # update symlink
 rm -f debian
-ln -s "packaging/${ID}-${VERSION_ID}" debian
+ln -s "packaging/${ID}-${VERSION_ID}/" debian
```

Comment:
this fixes the issue only if debian is not changed,  isn't the fix to generate the version before calling generate-packaging-dir which probably means supporting passing the changelog file to mkversion.sh if needed?

Reply:
Indeed, good point. I moved the version code up now.